lib/node_modules/@stdlib/blas/base/sdsdot/examples/index.js

```diff
@@ -18,23 +18,15 @@
 
 'use strict';
 
-var randu = require( '@stdlib/random/base/randu' );
-var round = require( '@stdlib/math/base/special/round' );
-var Float32Array = require( '@stdlib/array/float32' );
-var sdsdot = require( './../lib' ).ndarray;
+var discreteUniform = require( '@stdlib/random/base/discrete-uniform' ).factory;
+var filledarrayBy = require( '@stdlib/array/filled-by' );
+var sdsdot = require( '@stdlib/blas/base/sdsdot' );
 
-var x;
-var y;
-var i;
-
-x = new Float32Array( 10 );
-y = new Float32Array( 10 );
-for ( i = 0; i < x.length; i++ ) {
-	x[ i ] = round( randu()*100.0 );
-	y[ i ] = round( randu()*10.0 );
-}
+var x = filledarrayBy( 10, 'float32', discreteUniform( 0, 100 ) );
 console.log( x );
+
+var y = filledarrayBy( x.length, 'float32', discreteUniform( 0, 10 ) );
 console.log( y );
 
-var dot = sdsdot( x.length, 0.0, x, 1, 0, y, -1, y.length-1 );
+var dot = sdsdot( x.length, 0.0, x, 1, y, -1 );
```

Comment:
I see that you changed this example to match the README. That is okay. Better that they are consistent.